apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -841,7 +875,7 @@ export async function updateLti13Scores(
         scoreMaximum: 100,
         activityProgress: assessment_instance.open ? 'Submitted' : 'Completed',
         gradingProgress: 'FullyGraded',
-        userId,
+        userId: ltiUser.user_id,
```

Comment:
Man, having this `user_id` string keeps tripping me up, I was thinking this was our own `user_id` but it's actually the `sub`, right?

Reply:
Correct, the ltiUser is the ContextMembershipSchema directly from LTI, which uses `user_id` for what's called `sub` in other places.